components/tool/taskManager/src/hooks/useState.js

```diff
@@ -61,7 +61,10 @@ const useState = ({onCompleteAllTasks}) => {
         next(result)
       })
 
-  const checkIfAllTasksHaveBeenFinished = state => {
+  const checkIfAllTasksHaveBeenFinished = (taskId, state) => {
+    // No notice on not-visible tasks
```

Comment:
not sure what does this mean... _no notification_? 🤔

Reply:
Yep, probably using a bad word.

Reply:
This means that callback won't be executed